docs/source/spec/core.rst

```diff
@@ -4045,6 +4045,112 @@ settings from a service.
             }
         }
 
+The values for ``outputToken`` and ``items`` are paths. Paths are a series of
```

Comment:
Why do we not support paths for inputToken and pageSize?

Reply:
We can always add support for this later, but it wasn't added because it makes the operations significantly more cumbersome to use. For reference, 0 of the several hundred existing AWS paginators use paths for input.

Reply:
👌